src/ResourceManager/Reservations/Commands.Reservations/Reservations/AzureReservationAPIClient.cs

```diff
@@ -0,0 +1,867 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 1.2.2.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.Reservations
```

Comment:
This generated code should come from the management client nuget package

Reply:
fixed